backend/app/api/routes/chat.py

```diff
@@ -229,3 +229,14 @@ def get_recommend_questions(session: SessionDep, chat_message_id: int):
 
     return get_chat_message_recommend_questions(session, chat_message)
 
+
+@router.post("/chat-messages/{chat_message_id}/recommended-questions", response_model=List[str])
+def refresh_recommended_questions(session: SessionDep, chat_message_id: int):
+    chat_message = chat_repo.get_message(session, chat_message_id)
+    if not chat_message or len(chat_message.content) == 0:
+        raise HTTPException(
+            status_code=HTTPStatus.NOT_FOUND, detail="Chat message not found"
+        )
+
+    remove_chat_message_recommend_questions(session, chat_message_id)
```

Comment:
This will cause empty recommended questions list when error occur at generating new questions

Reply:
The next time call we can leverage the get function to generate it again.